### PR TITLE
[TACHYON-1705] Add the Chinese version of documentation page "Key-Value-Store-API.md"

### PR DESCRIPTION
**Pull request by leohjwang**
Chinese version of the doc is added, minor changes in the English version.

### CODE REVIEW

File: docs/cn/Key-Value-Store-API.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+```java
+KeyValueSystem kvs = KeyValueSystem.Factory().get();
+```
+
+## 创建一个新的键值存储
+
+可以通过调用`KeyValueSystem#createStore(TachyonURI)`来创建一个新的键值存储。将返回一个writer用于后续
+加入键值对。可以参照下面的例子：
+
+```java
```
Comment (yupeng9) [inline docs/cn/Key-Value-Store-API.md]: This piece of code can be extracted into a snippet for sharing?

File: docs/cn/Key-Value-Store-API.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+分区的切换对于用户是透明的。
+* 写入的键应该是排了序的并且没有重复的键值。
+
+## 从存储中读取值
+
+可以通过调用`KeyValueSystem#openStore(TachyonURI)`来读取一个完整的键值存储。将返回一个reader用于后续
+基于键的值读取。可以参照下面的例子：
+
+```java
+KeyValueStoreReader reader = kvs.openStore(new TachyonURI("tachyon://path/kvstore/"));
```
Comment (yupeng9) [inline docs/cn/Key-Value-Store-API.md]: This piece of code can be extracted into a snippet for sharing?

File: docs/cn/Key-Value-Store-API.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+reader.get("300");
+// Close the reader on the store
+reader.close();
+```
+## 通过迭代器遍历存储中的键值对
+
+可以参照下面的例子：
+
+```java
+KeyValueStoreReader reader = kvs.openStore(new TachyonURI("tachyon://path/kvstore/"));
```
Comment (yupeng9) [inline docs/cn/Key-Value-Store-API.md]: This piece of code can be extracted into a snippet for sharing?

**Comment by yupeng9** [comment]
Thanks for the translation. Also could you paste a screenshot of the translated version?

**Comment by leohjwang** [comment]
Here is a screenshot from translated version
github

**Comment by yupeng9** [comment]
LGTM. Thanks for the translation!

**Pull request merged**